ansible/roles/aws/tasks/main.yml

```diff
@@ -1 +1,17 @@
----
\ No newline at end of file
+---
+- name: Install AWS CLI
+  pip:
+    requirements: "{{ role_path }}/data/requirements.txt"
+  become: true
+
+- name: Copy shadow_upload.sh to the docker container
+  shell: "docker cp {{ role_path }}/files/shadow_upload.sh {{ container_name }}:/usr/local/bin/shadow_upload.sh"
+
+- name: Copy customer.key to the docker container
+  shell: "echo {{ customer_key }} > {{ role_path }}/files/customer.key; docker cp {{ role_path }}/files/customer.key {{ container_name }}:/usr/local/bin/customer.key"
```

Comment:
it is not that good idea to change file system of existing role. Please use /tmp for that

Reply:
Fixed in the last commit